install/prepare_compute_nodes.md

```diff
@@ -1,11 +1,17 @@
 # Prepare Compute nodes
 
-Some compute nodes types have special preparation steps, but most compute nodes are ready to be used now
-These nodes have an additional procedure before they can be booted.
+Some compute nodes types need to have additional procedures performed before
+they can be booted, but most compute nodes are ready to be used now.
 
 ### Topics:
```

Comment:
This whole section seems unnecessary if there is only 1 main procedure to follow.